internal/bminventory/inventory.go

```diff
@@ -4869,20 +4878,24 @@ func (b *bareMetalInventory) V2RegisterHost(ctx context.Context, params installe
 			params.NewHostParams.HostID.String(), params.InfraEnvID.String())
 		uerr := errors.Wrap(err, fmt.Sprintf("Failed to register host %s", hostutil.GetHostnameForMsg(host)))
 
-		// TODO event for infra-env
-		b.eventsHandler.AddEvent(ctx, params.InfraEnvID, params.NewHostParams.HostID, models.EventSeverityError,
-			uerr.Error(), time.Now())
+		// TODO: event for infra-env
+		// TODO: replace with proper InfraEnv event
```

Comment:
pls remove, it repeats of the previous comment line

Reply:
done